### PR TITLE
Fixes DEVELOPER-3479 (migration process)

### PR DESCRIPTION
**Pull request by LightGuard**
See https://issues.jboss.org/browse/DEVELOPER-3479 for full detail.

Currently, none of the pages are set to be ignored, you'll need to do that part yourself while testing.

### CODE REVIEW

**Comment by robpblake** [comment]
retest this please

**Comment by robpblake** [comment]
@LightGuard I'm just trying the migration process. I had the `/forum` page initially imported. I then added a new page at `/new-forums` and deleted the old forum page from Drupal and added the `ignore-export` directive. However when I try to move the page into the old `/forums` URL location, I get an error:

`The alias is already in use`

Have you been able to successfully re-use an alias?

**Comment by LightGuard** [comment]
retest this please

File: README.md [hunk 1]
```diff
@@ -375,6 +375,16 @@ If merge conflicts exist, you will need to do the fiollowing steps to fix the co
       
 5. Raise a PR from your branch onto the long running branch in upstream. Note that the PR tests will fail, as they don't expect a PR to be raised on a branch other than 'master'.
 
+## Migrating a page into Drupal
+
+1. Create the Drupal version of the page, but don't assign a URL alias (or assign a temporary alias)
+2. Review the Drupal version of the page
+3. Annotate the Awestruct version of the page by adding the `ignore_export: true` front matter variable to the page being exported. If the page being exported is an asciidoc page then it must be `:awestruct-ignore_export: true` instead. This will ensure that the Drupal export ignores the page and is not pushed into Drupal, whilst the legacy Awestruct CI job will still build the page.
+4. Delete the Awestruct pushed version of the page from Drupal
+5. Manually delete the old alias: Configuration -> URL aliases -> Find the alias you want to re-use and delete it
```
Comment (Dantheman720) [inline README.md]: Configuration -> Search and metadata -> URL Aliases

**Pull request merged**